### ISSUE TITLE
[Expo][Typescript] 'StyledView' cannot be used as a JSX component.

### ISSUE THREAD

**Issue by atanaskanchev** [fedemartinm/fast-styles#14]
**Describe the bug**
Typescript error on using styled component

**To Reproduce**
[Reproducible example](https://github.com/atanaskanchev/fs)

```tsx
import { styled } from "@fast-styles/react";
import { View } from "react-native";

const StyledView = () => {
  return styled(View, {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "red",
  });
};

export default StyledView;
```
<img width="1240" alt="image" src="https://github.com/fedemartinm/fast-styles/assets/4958587/44c34674-d8b6-424f-84fd-e81146afff17">

**Expected behavior**
No error expected.

**Environment**
    "@fast-styles/babel-plugin": "^0.2.3",
    "@fast-styles/react": "^0.2.9",
    "expo": "~49.0.8",
    "expo-status-bar": "~1.6.0",
    "react": "18.2.0",
    "react-native": "0.72.4"

**Comment by atanaskanchev** [comment]
Closing, I was wrongly defining the component as a return of an arrow function
`const StyledView = () => {
  return styled(View, {....`
should be 
`const StyledView = styled(View, {...`

**Comment by fedemartinm** [comment]
Got it. Some libraries that have HOCs that should be applied outside the render throw errors when the HOC is called improperly. I will try to add some kind of warning
